WcaOnRails/db/structure.sql

```diff
@@ -1014,6 +1014,7 @@ CREATE TABLE `registrations` (
   `deleted_at` datetime DEFAULT NULL,
   `deleted_by` int(11) DEFAULT NULL,
   `roles` text COLLATE utf8mb4_unicode_ci,
+  `assignments` text COLLATE utf8mb4_unicode_ci,
```

Comment:
We use JSON for several entities already (e.g. round results). Your point is definitely valid, I just thought we expect other apps to deal with showing this kind of stuff. Anyway I'm ok with implementing it the other way, just in case it pays off in the future. I've made an attempt in #3784, please check if that's more along the lines with your idea.